charts/komodor-agent/templates/node-enricher/_containers.tpl

```diff
@@ -9,6 +9,22 @@
   volumeMounts:
   - name: configuration
     mountPath: /etc/komodor
+  livenessProbe:
+    httpGet:
+      path: /healthz
+      port: http-healthz
+    periodSeconds: 60
+    initialDelaySeconds: 15
+    failureThreshold: 10
+    successThreshold: 1
+  readinessProbe:
+    httpGet:
+      path: /healthz
+      port: http-healthz
+    initialDelaySeconds: 5
+    periodSeconds: 5
+    failureThreshold: 3
+    successThreshold: 1
```

Comment:
The node-enricher currently does not expose a /healthz API - It's something we will need to add on the agent